AOSSIE-Org/Monumento
# Adding flutter Gen package ✅

## Description
Organized assets for seamless access through generated code.

### Motivation:
The integration of `flutter_gen` improves developer productivity and ensures reliable asset management.

Fixes #184 

## Type of change

## How Has This Been Tested?

- Ran `flutter pub run build_runner build` to generate the asset references.
- Verified that the auto-generated code correctly includes all assets.
- Tested asset usage in the app by replacing hardcoded asset paths with the `flutter_gen`-generated references.
- Ensured the app builds and runs without errors on both debug and release modes.

### Steps to Test:
1. Run `flutter pub run build_runner build` to generate the asset references.
2. Replace any asset path usages in the app with `Assets.<asset_path>`.
3. Run the app locally and ensure all assets (images, fonts, etc.) load correctly.

## Checklist:


## Maintainer Checklist


---

Let me know if you need any further refinements!
